### DISCUSSION TITLE
v1.1.5

### DISCUSSION THREAD

**Discussion by gabbybilka** [microsoft/WindowsAppSDK#2968] [Announcements]
## Windows App SDK 1.1.5 is now available

Windows App SDK 1.1.5 is a stable servicing release that fixes critical bugs for 1.1. 

- If you're building an app with the Windows App SDK for the first time, follow the [installation instructions here](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app?tabs=desktop-csharp).

- If you are already using the Windows App SDK 1.1 and would like to update to the latest release, you can do so by following the [update instructions here](https://docs.microsoft.com/windows/apps/windows-app-sdk/update-existing-projects-to-the-latest-release).

Check out our [release notes](https://aka.ms/windowsappsdk/1.1/1.1.5/release-notes) for the list of issues fixed in this release.

### Updating to this version

The 1.1.5 package is available at https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.1.5

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.1.5'>v1.1.5</a>.</em>

**Comment by yki5000** [comment]
Hello,
Could anybody give me a hint?
I've installed 1.1.5 Redistibutable and VS Extension, uninstalled everything (SDK 1) before and i still get this message (c++/ unpackaged app):

D:\a_work\1\s\dev\WindowsAppRuntime_BootstrapDLL\MddBootstrap.cpp(536)\Microsoft.WindowsAppRuntime.Bootstrap.dll!00007FF88F1C38F9: (caller: 00007FF88F1C244F) LogHr(1) tid(5034) 80040010 Object is not in any of the inplace active states
Msg:[Bootstrap.Intitialize: Scanning packages for Major.Minor=1.0, Tag=, MinVersion=2.460.358.0]
D:\a_work\1\s\dev\WindowsAppRuntime_BootstrapDLL\MddBootstrap.cpp(665)\Microsoft.WindowsAppRuntime.Bootstrap.dll!00007FF88F1C4C4C: (caller: 00007FF88F1C244F) Exception(1) tid(5034) 80070491 There was no match for the specified key in the index.
Msg:[Enumeration: Major.Minor=1.0, Tag=, MinVersion=2.460.358.0]
D:\a_work\1\s\dev\WindowsAppRuntime_BootstrapDLL\MddBootstrap.cpp(125)\Microsoft.WindowsAppRuntime.Bootstrap.dll!00007FF88F1D3E61: (caller: 00007FF7FA3F923E) ReturnHr(1) tid(5034) 80070491 There was no match for the specified key in the index.
Msg:[D:\a_work\1\s\dev\WindowsAppRuntime_BootstrapDLL\MddBootstrap.cpp(665)\Microsoft.WindowsAppRuntime.Bootstrap.dll!00007FF88F1C4C4C: (caller: 00007FF88F1C244F) Exception(1) tid(5034) 80070491 There was no match for the specified key in the index.
Msg:[Enumeration: Major.Minor=1.0, Tag=, MinVersion=2.460.358.0]

**Reply by DarranRowe** [reply]
Well, the biggest hint is the major/minor version. Notice in what you posted there, there is Major.Minor=1.0?

When built against the Windows App SDK 1.1.5, this output in the debug output is:

```
D:\a\_work\1\s\dev\WindowsAppRuntime_BootstrapDLL\MddBootstrap.cpp(814)\Microsoft.WindowsAppRuntime.Bootstrap.dll!00007FFA15FEA829: (caller: 00007FFA15FE937F) LogHr(2) tid(1854) 80040010 Object is not in any of the inplace active states
    Msg:[Bootstrap.Intitialize: Scanning packages for Major.Minor=1.1, Tag=, MinVersion=1005.616.1651.0] CallContext:[\Initialize] 
```
Notice here how the major/minor version is 1.1?

The application you are trying to debug/run was built with the Windows App SDK 1.0. This means that you have to install the 1.0.4 redistributable for that application to run. To put it simply, the major and minor version numbers (the first two numbers) have to match EXACTLY for the redistributable to satisfy the requirements.

If you want/need to use features in version 1.1 then you must update the Windows App SDK nuget package to a 1.1 version.

**Reply by yki5000** [reply]
Great, thank you very much!
